assets/src/BlockFilters.js

```diff
@@ -23,3 +24,19 @@ const addFileBlockFilter = () => {
 };
 
 const addImageBlockFilter = () => addFilter('editor.BlockEdit', 'core/image/edit', ImageBlockEdit);
+
+// Enforce "AJAX" toggle setting enabled by default, on Gravity form block.
+const addGravityFormsBlockFilter = () => {
+  const setAJAXToggleDefualtTrue = (settings, name) => {
```

Comment:
if you can just fix a typo inverting `a` and `u` in `Default`, to `setAJAXToggleDefualtTrue` => `setAJAXToggleDefaultTrue` (and use the same on l.41) :pray:

Reply:
my bad 🙃 Fixed 👍